apps/ui/src/helpers/format.ts

```diff
@@ -0,0 +1,3 @@
+export function removeTrailingZeroes(value: number, maxDecimals: number) {
```

Comment:
This is new file instead of `utils` as I think we should start moving all formatting related functions out of very generic `utils` in the future to better organize the code.